src/civis/futures.py

```diff
@@ -151,16 +143,38 @@ def _exception_from_logs(self, exc, nlog=15):
 
     @property
     def job_id(self):
+        """The job ID for the Civis Platform job that this future is tracking.
+
+        Returns
+        -------
+        int
+        """
         return self.poller_args[0]
 
     @property
     def run_id(self):
+        """The run ID for the Civis Platform job that this future is tracking.
+
+        Returns
+        -------
+        int | None
+        """
         try:
             return self.poller_args[1]
         except IndexError:
             # when poller function has job_id only but not run_id
             return None
 
+    @property
+    def job_url(self):
+        """The URL for the Civis Platform job that this future is tracking.
+
+        Returns
+        -------
+        str
+        """
+        return f"https://platform.civisanalytics.com/spa/#/jobs/{self.job_id}"
```

Comment:
If one takes a rendered URL from here and views it in a web browser, Civis Platform will correctly redirect from "jobs" to the "scripts/<script_type>" URL, which is nice.